### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Marcraphael12**
### Project requirements

- You will find all the details of the Bookstore design in [Zeplin](https://zeplin.io/).
- [Login to Zeplin](https://app.zeplin.io/login) (hint: you will find the credentials in the [sneak peek](./sneak_peek.md) of the project).
- Open the project *Bookstore CMS*.
- Use the information detailed in Zeplin to style the website.
- Your final design should match Zeplin's design.

### CODE REVIEW

File: src/style/base/App.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.app {
```
Comment (elmar8287) [inline src/style/base/App.css:1]: - [ ] Please, try to fix some points to be closer to the original styling:

1. Please, try fixing the width of the navbar. It should be as in original styling, cover the whole page width, not the same with the books list section.
2. Try to fix here the padding, make it closer to the original version 👍 
3. Please, try to make this section closer to the original one - use `font-family: RobotoSlab;` and fix the color where needs 👍 
4. The "Add book" button needs to be styled as in the original version, Here are the original options: 
```
width: 11.5rem;
height: 2.813rem;
margin: 1.188rem 0 0 2.125rem;
padding: 0.801rem 1.188rem 0.886rem 1.375rem;
border-radius: 3px;
background-color: var(--azure); 
```
5. The border color for each added book should be grayer `border: solid 1px var(--grey-border);`

Screen Shot 2022-04-02 at 22 59 46

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- No linter issues 👍 
- Nice commit history 👍 
- Informative PR description 💯 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- Please, try in the future to give names for commits with capital letters 👍 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style/base/App.css (hunk 1 above)
Comment (Marcraphael12) [reply src/style/base/App.css:1]: I've tried my best to apply them... thanks

**Review by bimbolabuari — APPROVED** [review]
Hi @Marcraphael12,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**